interfaces/cython/cantera/constants.pyx

```diff
@@ -12,6 +12,92 @@ cdef extern from "cantera/base/ct_defs.h" namespace "Cantera":
     cdef double CxxPermeability_0 "Cantera::permeability_0"
     cdef double CxxEpsilon_0 "Cantera::epsilon_0"
 
+
+#: Get an element's molar mass
+#: Deuterium and Tritium can be looked up by name or symbol,
+#: but cannot be found by atomic number
+def element_weight(element):
+    if isinstance(element, (str, unicode, bytes)):
+        return getElementWeight(stringify(element))
+    elif isinstance(element, int):
+        return getElementWeight(<int>element)
+
+
+#: Get the number of elements defined in Cantera, including Deuterium and Tritium
+def num_elements_defined():
+    return numElementsDefined()
+
+
+#: Get a list of all of the true element weights defined in Cantera
+#: Deuterium and Tritium are also defined, but not included in this list
+def element_weights():
+    return [element_weight(m+1) for m in range(num_elements_defined())]
+
+
+#: Get an element's name from its atomic number
+#: Deuterium and Tritium are also defined, but not included in this function
+def element_name(element):
+    if isinstance(element, (str, unicode, bytes)):
+        return pystr(getElementName(stringify(element)))
+    elif isinstance(element, int):
+        return pystr(getElementName(<int>element))
+
+
+#: Get a list of the true element names defined in Cantera
+#: Deuterium and Tritium are also defined, but not included in this list
+def element_names():
+    return [element_name(m+1) for m in range(num_elements_defined())]
+
+
+#: Get an element's symbol from its atomic number
+def element_symbol(element):
+    if isinstance(element, (str, unicode, bytes)):
+        return pystr(getElementSymbol(stringify(element)))
+    elif isinstance(element, int):
+        return pystr(getElementSymbol(<int>element))
+
+def atomic_number(element):
+    return getAtomicNumber(stringify(element))
+
+#: Get a list of the true element symbols defined in Cantera
+#: Deuterium and Tritium are also defined, but not included in this list
+def element_symbols():
+    return [element_symbol(m+1) for m in range(num_elements_defined())]
+
+class Element(object):
+    """
+    """
+    def __init__(self, arg):
+        if isinstance(arg, (str, unicode, bytes)):
+            try:
```

Comment:
In the try/except, we only end up with one of name and symbol defined. Need to assign `arg` to the other one.